src/Pages/homepage.jsx

```diff
@@ -1,4 +1,5 @@
 import { gql, useQuery } from "@apollo/client";
+import { Link } from "react-router-dom";
 
 const pokeQuery = gql`
   query samplePokeAPIquery {
```

Comment:
I know this line is not changed in this PR but we can probably rename this query too like the one I pointed out in the `<PokemonDetail>` component.

```suggestion
  query pokemonQuery {
```